python/GafferArnoldTest/ArnoldRenderTest.py

```diff
@@ -1094,13 +1094,19 @@ def testOSLShaders( self ) :
 		swizzle.loadShader( "MaterialX/mx_swizzle_color_float" )
 		swizzle["parameters"]["in"].setValue( imath.Color3f( 0, 0, 1 ) )
 		swizzle["parameters"]["channels"].setValue( "b" )
+		outPlug = swizzle["out"]["out"]
 
-		pack = GafferOSL.OSLShader()
-		pack.loadShader( "MaterialX/mx_pack_color" )
-		pack["parameters"]["in1"].setInput( swizzle["out"]["out"] )
+		# mx_pack_color doesn't ship with OSL 1.11, nor is it required any longer
+		# since we don't have any easy with in python to query OSL version, we
+		# assume a missing shader indicates we're on a new enough OSL version.
+		with IECore.IgnoredExceptions( RuntimeError ) :
+			pack = GafferOSL.OSLShader()
+			pack.loadShader( "MaterialX/mx_pack_color" )
+			pack["parameters"]["in1"].setInput( outPlug )
+			outPlug = pack["out"]
 
 		ball = GafferArnold.ArnoldShaderBall()
-		ball["shader"].setInput( pack["out"] )
+		ball["shader"].setInput( outPlug )
```

Comment:
Can we use another shader here instead (one that exists for both OSL versions)? We're not specifically trying to test `mx_pack_color`, but we are trying to test that we can render a network of shaders. Having just one shader means we're not testing connections any more.

Reply:
Addressed in 22a627dc4f68de1479127bc606909f2d30b44f90